pbxproj/pbxsections/PBXBuildFile.py

```diff
@@ -35,7 +35,11 @@ def _print_object(self, indentation_depth=u'', entry_separator=u'\n', object_sta
                                                        indentation_increment=u'')
 
     def _get_comment(self):
-        return u'{0} in {1}'.format(self.fileRef._get_comment(), self._get_section())
+        if hasattr(self, 'fileRef'):
```

Comment:
For the sake of compression, the else can be removed, by initializing the comment as follow:
```
comment = "(null)"
if hasattr(self, 'fileRef'):
    comment = self.fileRef._get_comment()
```